addons/info/src/components/PropTable.js

```diff
@@ -137,34 +103,34 @@ export default function PropTable(props) {
   };
 
   return (
-    <table style={stylesheet.propTable}>
+    <table style={styles.propTable}>
```

Comment:
Not a requirement for this PR, but I'd really really like it if we could move the styling components into `lib/components`. and style them using glamorous. Styling using inline-styles give pretty poor development experience and makes it very hard for us to add theming support in the long run.

Again: not a requirement to get this merged, just a head's up if we don't do this in this PR, it will be done in a refactor later.